README.md

```diff
@@ -1,185 +1,80 @@
 # ecDNA-Echo
 Pipeline to Analyze ecDNA in collaboration with BoundlessBio
 
-## Version 2
+### Dependencies
 
-### Building Echo-Preprocessor Image
+The environment yml file for the scripts may be found in ```/envs/echo.yml```
+The environment yml file for the analysis notebooks may be found in ```/envs/ecDNA_analysis.yml```
 
-#### Singularity
+You can get all the dependencies for the scripts with 
 
 ```
-export singularity_cache=$HOME/.singularity/cache
-
-echo $singularity_cache
-
-singularity build --docker-login  ${singularity_cache}/boundlessbio-echo-preprocessor-v2.0.4.img docker://boundlessbio/echo-preprocessor:release-v2.0.4
+conda env create --name ecDNA --file=envs/echo.yml
+conda activate ecDNA
+pip install git+https://github.com/mskcc/facetsAPI#facetsAPI
```

Comment:
we can add a note here or at the bottom of readme "if there are issues installing the facetsapi, please make sure that you are using the Pip3 version or point it to the version of pip inside the conda env"